### ISSUE TITLE
Fails to build from crates.io

### ISSUE THREAD

**Issue by jayvdb** [sile/jaegercat#2]
```
> cargo install jaegercat
    Updating crates.io index
  Downloaded jaegercat v0.1.1
  Downloaded 1 crate (6.8 KB) in 0.98s
  Installing jaegercat v0.1.1
    Updating crates.io index
error: failed to compile `jaegercat v0.1.1`, intermediate artifacts can be found at `/tmp/cargo-install6EioiQ`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.

Caused by:
  failed to select a version for the requirement `rmp-serde = "^0.13"`
  candidate versions found which didn't match: 1.3.0, 1.3.0-beta.1, 1.2.0, ...
  location searched: crates.io index
  required by package `serdeconv v0.3.0`
      ... which satisfies dependency `serdeconv = "^0.3"` of package `jaegercat v0.1.1`
  if you are looking for the prerelease package it needs to be specified explicitly
      rmp-serde = { version = "1.3.0-beta.1" }
  perhaps a crate was updated and forgotten to be re-vendored?
```

**Comment by sile** [comment]
Hmm, I'll look into this issue within a few days.

**Comment by sile** [comment]
I have just released jaegercat-0.2.0, and confirmed that the latest version can be built from crates.io.
